app/services/waste_carriers_engine/notify/registration_activated_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  module Notify
+    class RegistrationActivatedService < BaseSendEmailService
```

Comment:
Should we clarify that this is an email-sending service in the name, eg RegistrationActivatedEmailService? Since we're still working out if a confirmation letter is needed as well as the email, there is the chance of us needing a RegistrationActivatedLetterService or something similar in the future.

Reply:
Good idea. My initial thought was that this would be assumed by the Notify scope. But Notify isn't just for emails!